TheBTeam.ConsoleApp/MainMenu.cs

```diff
@@ -100,20 +101,26 @@ public static void ShowMainMenu()
                     TransactionViewer.ViewTransaction(tmpListTransactions.TransactionsList);
                     Console.ReadKey();
                 }
-                else if (mainMenuItem[currentItem] == ("Show transaction according Category"))
+                else if (mainMenuItem[currentItem] == ("Show transaction by Category"))
                 {
                     Console.WriteLine($"{mainMenuItem[currentItem]}");
                     TransactionViewer.ViewTransactionAccordingCategory(tmpListTransactions.TransactionsList);
                     Console.ReadKey();
                 }
+                else if (mainMenuItem[currentItem] == "Show Users transaction by Type")
+                {
+                    Console.WriteLine($"{mainMenuItem[currentItem]}");
+                    TransactionViewer.UserAndTypeTransaction(tmpListUsers.UsersList, tmpListTransactions.TransactionsList);
+                    Console.ReadKey();
+                }
                 else if (mainMenuItem[currentItem] == ("Edit existing user"))//there used to be like 3 paragraphs here regarding my reasoning, but fuck that, basically edit user wont work once we pass a tmplist
-                    //to it directly its messy due to the way references are handled
-                    //or maybe more the way i think they are handled - lists being copied, and objects referred
-                    //bottom line is it works but there definitely is to be a nicer way to do this, ill ask patryk during refinements
-                    //Actually nvm im dumb, since below works i jsut understood sb wrong, ill get below to look decent when im done with homework
+                                                                             //to it directly its messy due to the way references are handled
+                                                                             //or maybe more the way i think they are handled - lists being copied, and objects referred
```

Comment:
Feel free to yeet the comment, it keeps getting added back whenever its present in sbs branch i think, logic applied turned out to actually make sense, althouh in an entirely different way, its obsolete